README.md

```diff
@@ -58,21 +64,21 @@ During command run you will see log messages with transfer report. For example:
 
 ### Attention!
 
-This program **will not** change your posts content. All `src` attributes of images in posts will stay as is! 
-Images in user-part of wordpress site will be changed "on the fly", but only if you use wordress theme with wordpress Gutenberg blocks and adaptive delivery in plugin settings is on.
+This program **will not** change your posts content. All `src` image attributes will stay unchanged.
+Images in the user-part of a WordPress site will be changed on the fly. The only requirement is that you use WordPress Gutenberg blocks and an Adaptive Delivery in Uploadcare plugin's settings is on.
 
 ## Development
 
-This is a Symfony single-command application, uses a doctrine/dbal for database interaction. All application logic is in `Uploadcare\WpMigrate\Command\MigrateCommand` class.
+This is a Symfony single-command application, it uses a `doctrine/dbal` for database interaction. All application logic is in the `Uploadcare\WpMigrate\Command\MigrateCommand` class.
 
-To make new phar-archive (in case you changed something) download [Phar-composer](https://github.com/clue/phar-composer) package phar archive: `curl -JOL https://clue.engineering/phar-composer-latest.phar`. 
+To make a new PHP archive (in case you changed something), download [phar-composer](https://github.com/clue/phar-composer). Create the phar: `curl -JOL https://clue.engineering/phar-composer-latest.phar`. 
```

Comment:
"Phar-archive" is a common term that uses in php-community. Please, leave it as is.

Reply:
okay i thought one would read it PHP archive archive, but phar archive is present out there, i checked too.